java/cdk/src/main/java/sleeper/cdk/stack/TransactionLogSnapshotStack.java

```diff
@@ -87,7 +89,7 @@ private void createSnapshotCreationLambda(InstanceProperties instanceProperties,
                 .runtime(Runtime.JAVA_11)
                 .handler("sleeper.statestore.snapshot.TransactionLogSnapshotCreationTriggerLambda::handleRequest")
                 .environment(Utils.createDefaultEnvironment(instanceProperties))
-                .reservedConcurrentExecutions(1)
+                .reservedConcurrentExecutions(instanceProperties.getInt(TRANSACTION_LOG_SNAPSHOT_DELETION_LAMBDA_CONCURRENCY_RESERVED))
```

Comment:
This should be set on the snapshot creation function. The trigger function needs this set to 1.

This should also use the snapshot creation property, rather than the snapshot deletion property.

Reply:
Adressed